source/nodejs/adaptivecards-designer/src/adaptivecards-designer.ts

```diff
@@ -17,3 +18,8 @@ export const defaultMicrosoftHosts: containers.HostContainer[] = [
 	new containers.CortanaClassicContainer("Cortana Skills (Classic)", "containers/cortana-classic-container.css"),
 	...containers.BerlinContainer.allContainers(),
 ]
+
+export const defaultMicrosoftDeviceEmulations: DeviceEmulation[] = [
+	{ name: 'Unconstrained', styleSheet: 'unconstrained.css' },
```

Comment:
I'm not sure I understand the reason for a device-emulation-specific CSS when the device-emulation itself is disconnected from the HostContainer.

The HostContainer implementation is the entity responsible for emulating a device's look and feel, and it has its own CSS. What's needed on top of that is emulating the screen width of a device, but that shouldn't change the device's look and feel.

When would this `styleSheet` be applied, and what does it control?

Reply:
Moot per subsequent commits.